proxmox/cluster/acme/plugins/acme_plugins_types.go

```diff
@@ -0,0 +1,135 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+package plugins
+
+import (
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"net/url"
+	"strings"
+)
+
+// ACMEPluginsListResponseBody contains the body from a ACME plugins list response.
+type ACMEPluginsListResponseBody struct {
+	// Unique identifier for ACME plugin instance.
+	Data []*ACMEPluginsListResponseData `json:"data,omitempty"`
+}
+
+// ACMEPluginsListResponseData contains the data from a ACME plugins list response.
+type ACMEPluginsListResponseData struct {
```

Comment:
Some of these types are very similar, should we combine them or leave them as is in case future updates cause divergence?

Reply:
I would extract the common set of fields into a "base" struct, and extend it in specific `*Data` structs.